sample/src/main/java/io/instaleap/statusprogressbar/MainViewModel.kt

```diff
@@ -0,0 +1,127 @@
+package io.instaleap.statusprogressbar
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+
+class MainViewModel : ViewModel() {
+
+    private val _dataModelView = MutableLiveData<DataModelView>()
+    val dataModelView: LiveData<DataModelView> = _dataModelView
+
+    private val _dataList = MutableLiveData<List<DataModelView>>()
+    val dataList: LiveData<List<DataModelView>> = _dataList
+
+    private val _pendingQuantity = MutableLiveData(10)
```

Comment:
Why this default value?

Reply:
I'm going to change to constat val. This is to simulate the total items in the order, and just set as Pending.